### ISSUE TITLE
Favicon Replacement issue

### ISSUE THREAD

**Issue by Inrx** [filipecarneiro/hugo-bootstrap-theme#5]
Hey Filipe, love this theme. 
I'm currently trying to change the Favicon. 
1) I just replaced the original favicon files with my owns but it didn't work. Kept `favicons.html` untouched. Result: No favicon at all. 
2) Tried to use `realfavicongenerator.net` as suggested and added the files to the `static` dir. Replaced the html code with the suggested by `realfavicongenerator.net` to no avail. 

I have no experience in this, is there something I am missing? Thanks

**Comment by filipecarneiro** [comment]
Hi Inês,

Thank you for your feedback!

Just replacing the favicons files in the static folder should work… lets confirm a couple of things:

1. You have placed `favicon.svg` and/or `favicon.png` in `static` folder;
2. You didn’t override any of the base templates for this (`_default/baseof.html`, `head/head.html` and `head/favicons.html`) or, if you did, you have included this path, right?

Please run a local build with `–cleanDestinationDir`, using `npm run build` or `hugo –cleanDestinationDir`

Are favicons files placed on the root of your `public` folder?

Also, check index.html in your public folder and look for `<link rel="icon" type=… href=…>` just before `</head>`

Are those lines there? Is the `href` path valid?

**Comment by Inrx** [comment]
Hey there, Filipe.

Thanks for your help. 
There were two problems:
1) I didn't know I had to run `hugo` before `hugo server` so, in reality, the public folder was never created. Because of this the browser never assumed the favicon change. 
2) Because the new favicons were never assumed, I did change the `favicons.html` to see if I managed to find the problem. In the process I broke it. 

So the solution was just to copy paste the `favicons.html` from the Theme directory, run `hugo` and voilá!.

Thanks again, 
Inês
